### ISSUE TITLE
Keywords: categorie veranderd, maar nog steeds zichtbaar

### ISSUE THREAD

**Issue by LindsayJanssen** [martijnbentum/heritages#123]
Hoi Martijn,

Ik ben onze keyword en category lijst aan het opschonen. Daarbij wilde ik de term "soup kitchens" verplaatsen van de category "food" naar de category "relief". Het resultaat is echter dat "soup kitchens" nu bij beide categorieën lijkt te staan, maar toch ook weer niet? Wanneer ik naar dit deel van het category scherm kijk:

![image](https://user-images.githubusercontent.com/69973344/157396278-dc03d78c-f3d3-4271-879f-c90a7b0fa9ca.png)

Zie ik "soup kitchens" nog steeds genoemd staan. Maar wanneer ik dan naar beneden scroll om de onder elkaar opgesomde lijst keywords te bekijken, staat soup kitchens daar niet meer. Weet jij hoe dit nu zit? Ik vermoed dat ik hier vaker tegenaan ga lopen, omdat ik voor meer keywords en categories aanpassingen moet maken (niet ideaal, ik weet het).

Alvast veel dank en hartelijke groet!

Lindsay

**Comment by LindsayJanssen** [comment]
Update: ik heb een lijst gemaakt van alle items die "soup kitchens" als keyword hadden. Daarna heb ik het keyword verwijderd, en nogmaals een nieuw keyword "soup kitchens" -- maar nu onder de categorie "relief" -- aangemaakt. Vervolgens heb ik de items opnieuw getagd met de term.

Volgens mij ging het niet goed wanneer ik poogde de termen Food/soup kitchens en Relief/soup kitchens naast elkaar te laten bestaan. Beide termen (Food en Relief) werden dan weergegeven bij een item, maar ik kon niet een van de twee verwijderen.

**Comment by martijnbentum** [comment]
Ik weet niet zo goed wat er met soup kitchens aan de hand was. Als ik probeer de categorie van een keyword te vervangen dan werkt dat. Het zou inderdaad te maken kunnen hebben met dat het bij beide termen gekoppeld was.

Is er iets dat veranderd moet worden?

**Comment by LindsayJanssen** [comment]
Nee, dat denk ik niet. Excuses dat dit een nodeloze oefening was in mijn denkproces;-)
